schematic/schemas/explorer.py

```diff
@@ -649,46 +681,48 @@ def edit_class_nx(self, class_mod: dict) -> None:
         # set the networkx schema graph to the the modified networkx schema
         self.schema_nx = schema_graph_nx
 
+        # print("Added node {} to the graph successfully.".format(schema_object["rdfs:label"]))
+
         # part of the code that replaces the modified class in the original JSON-LD schema (not in the data/ folder though)
         for i, schema_class in enumerate(self.schema["@graph"]):
-            if schema_class["rdfs:label"] == class_mod["rdfs:label"]:
-                validate_class_schema(class_mod)    # validate that the class to be modified follows the structure for any generic class (node)
+            if schema_class["rdfs:label"] == schema_object["rdfs:label"]:
+                # validate_class_schema(schema_object)    # validate that the class to be modified follows the structure for any generic class (node)
```

Comment:
Right, so the purpose of this new parser is to improve time complexity/the amount of time it takes to generate a JSON-LD schema from the CSV file. And we want to avoid any operation that could add to this time taken.

Since the JSON-LD isn't being modified manually, we needn't validate the class each and and every time before adding it to the graph (and the JSON-LD schema object). The validation will become necessary only when a manually edited class (or property) is being added/updated.

Reply:
@sujaypatil96  +1

Reply:
That makes sense. Should we then remove all parts of the code that are commented out, or should we nest them in an if conditional that defaults to false based on a function argument?

Reply:
This makes sense - I'd vote for argument-based logic. But I think it's fine to keep the function calls commented out in this PR; we can update later.

Reply:
Sounds good to me. @sujaypatil96: make sure to open an issue if you wish to tackle this in a separate issue so we don't forget.